apps/ok/lib/ok.ex

```diff
@@ -0,0 +1,128 @@
+defmodule Ok do
+  @moduledoc """
+  Ok provides functions for handling and returning `{:ok, term}` tuples,
```

Comment:
-_-

Reply:
Looks like someone had too much time on their hands

Reply:
Yea, looks like there is a library called Brex_result  which does the same thing and in /hindersight/ they wished they used.  That said, their libraries as they are rely on this library so thats another refactor.